src/server/implementation/player/cmd/inventory.c

```diff
@@ -22,6 +22,8 @@ int get_inventory(data_t *data, __attribute__((unused)) char **args)
         data->clients[data->curr_cli_index]->player->inventory[4],
         data->clients[data->curr_cli_index]->player->inventory[5],
         data->clients[data->curr_cli_index]->player->inventory[6]);
+    if (value == -1)
+        return 1;
```

Comment:
We can let this but remember that `ko` is not sent at the execution time anymore.
It can only be thrown at schedule time.